src/synthesizer/emission_models/stellar/models.py

```diff
@@ -78,26 +80,71 @@ class LineContinuumEmission(StellarEmissionModel):
     def __init__(
         self,
         grid,
-        label="linecont",
+        label="nebular_line",
         fesc=0.0,
+        fesc_ly_alpha=1.0,
         **kwargs,
     ):
         """
-        Initialise the LineContinuumEmission object.
+        Initialise the NebularLineEmission object.
 
         Args:
             grid (synthesizer.grid.Grid): The grid object to extract from.
+            fesc_ly_alpha (float): The escape fraction of Lyman-alpha.
             label (str): The label for this emission model.
             fesc (float): The escape fraction of the emission.
         """
-        StellarEmissionModel.__init__(
-            self,
-            grid=grid,
-            label=label,
-            extract="linecont",
-            fesc=fesc,
-            **kwargs,
-        )
+        if fesc_ly_alpha < 1.0:
```

Comment:
The way this has always been used previously had this logic in the `NebularEmission`, has this changed? Is there a situation when you would be using this model without `fesc_ly_alpha=1`? If not we can simplify things.

Reply:
if you want the lines, you need to know what the lyman alpha escape fraction is so you can modify the luminosity of that line, so yes it needs to be here. Arguably it only needs to be here, since the nebular emission should derive from the combination of the lines and continuum

Reply:
Sounds good to me. Lets kill the logic in the `NebularEmission` model then. Just simplifies things a bit if we don't have multiple of the same branching logic.